gossipd/routing.c

```diff
@@ -1368,6 +1368,19 @@ bool routing_add_channel_update(struct routing_state *rstate,
 		/* Maybe announcement was waiting for this update? */
 		uc = get_unupdated_channel(rstate, &short_channel_id);
 		if (!uc) {
+			/* Allow ld to process a private channel update */
+			struct remote_priv_update remote_update;
+			u8* msg;
+			remote_update.source_node = peer->id;
+			remote_update.scid = short_channel_id;
+			remote_update.fee_base = fee_base_msat;
+			remote_update.fee_ppm = fee_proportional_millionths;
+			remote_update.cltv_delta = expiry;
+			remote_update.htlc_minimum_msat = htlc_minimum;
+			remote_update.htlc_maximum_msat = htlc_maximum;
+			msg = towire_gossipd_remote_channel_update(NULL, &remote_update);
+			daemon_conn_send(peer->daemon->master, take(msg));
+
```

Comment:
There are now two of these in gossipd??  Unify into a single ... `tell_lightningd_private_cupdate(id, scid, base, ppm, expiry, min, max).`?